src/System.Security.Cryptography.Algorithms/src/System/Security/Cryptography/ECDsa.cs

```diff
@@ -8,6 +8,20 @@ namespace System.Security.Cryptography
 {
     public abstract class ECDsa : AsymmetricAlgorithm
     {
+        public ECDsa() { }
+
+        public ECDsa(ECNamedCurve curve)
+        {
+            if (curve.Oid != null)
+            {
+                _curve = new Oid(curve.Oid.Value, curve.Oid.FriendlyName);
+            }
+        }
+
+        public abstract ECParameters ExportNamedCurveParameters(bool includePrivateParameters);
```

Comment:
No magic here. If you add an abstract member it must be added to all the places supporting the new contract version. It can either be abstract or virtual/throws in the implementation.